rss/routes.py

```diff
@@ -0,0 +1,28 @@
+"""URL routes for RSS feeds"""
+
+from flask.json import jsonify
+from flask import Blueprint
+from werkzeug.exceptions import NotFound
+from arxiv import status
+
+blueprint = Blueprint('rss', __name__, url_prefix='')
+
+@blueprint.route('/rss', methods=['GET'])
+def bare_rss() -> None:
+    """
+    Disallow bare requests.
+
+    :return:
```

Comment:
Fine for now; as you go forward, we are using [Numpy-style docstrings](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard), which do param/return/raises statements a bit differently.